src/SELF_DomainDecomposition_t.f90

```diff
@@ -275,25 +240,25 @@ subroutine FinalizeMPIExchangeAsync(mpiHandler)
 
   endsubroutine FinalizeMPIExchangeAsync
 
-  subroutine GlobalReduce_RealScalar(mpiHandler,sendBuf,recvBuf)
-    class(MPILayer_t),intent(in) :: mpiHandler
-    real(prec),intent(in) :: sendBuf
-    real(prec),intent(out) :: recvBuf
-    ! Local
-    integer :: iError
-
-    if(mpiHandler%mpiEnabled) then
-      call MPI_ALLREDUCE(sendBuf, &
-                         recvBuf, &
-                         1, &
-                         mpiHandler%mpiPrec, &
-                         MPI_SUM, &
-                         mpiHandler%mpiComm, &
-                         iError)
-    else
-      recvBuf = sendBuf
-    endif
-
-  endsubroutine GlobalReduce_RealScalar
-
-endmodule SELF_MPI_t
+  ! subroutine GlobalReduce_RealScalar(mpiHandler,sendBuf,recvBuf)
```

Comment:
This commented out bit reminds me that we need to set up parallel reduction for the entropy calculation methods.

Reply:
Resolved with [4f7a7a0](https://github.com/FluidNumerics/SELF/pull/53/commits/4f7a7a0e157b85dc677092583d328816442ade81)